### PR TITLE
Fastly IP Ranges (ipv6_cidr_blocks)

### PR DESCRIPTION
**Pull request by ghost**
Updated `go-fastly` to `v1.14.0`. Exposed `ipv6_cidr_blocks` on data source. Updated test. Updated documentation.

### CODE REVIEW

File: website/docs/d/ip_ranges.html.markdown [hunk 1]
```diff
@@ -19,16 +19,18 @@ resource "aws_security_group" "from_fastly" {
...
+    protocol          = "tcp"
+    cidr_blocks       = ["${data.fastly_ip_ranges.fastly.cidr_blocks}"]
+    ipv6_cidr_blocks  = ["${data.fastly_ip_ranges.fastly.ipv6_cidr_blocks}"]
   }
 }
 ```
 
 ## Attributes Reference
 
 * `cidr_blocks` - The lexically ordered list of CIDR blocks.
```
Comment (phamann) [inline website/docs/d/ip_ranges.html.markdown]:
```suggestion
* `cidr_blocks` - The lexically ordered list of ipv4 CIDR blocks.
```

**Review by phamann — APPROVED** [review]
Thank you for the contribution @wbautistaEXPEDIA. Looks good to me other than the small docs update.

File: website/docs/d/ip_ranges.html.markdown (hunk 1 above)
Comment (ghost) [reply website/docs/d/ip_ranges.html.markdown]: @phamann 
Good catch. Fixed.

**Comment by ghost** [comment]
The CI check appears to be stale. The build was successful.

**Comment by phamann** [comment]
@wbautistaEXPEDIA This was released in v0.16.0 of the provider. Thank you for your contribution!

**Pull request merged**